ldes-server-domain/src/test/java/be/vlaanderen/informatievlaanderen/ldes/server/domain/ldes/eventstream/services/EventStreamConverterImplTest.java

```diff
@@ -22,10 +24,11 @@ class EventStreamConverterImplTest {
 
 	private final PrefixAdder prefixAdder = new PrefixAdderImpl();
 	private EventStreamConverter eventStreamConverter;
+	private LdesConfig ldesConfig;
 
 	@BeforeEach
 	void setUp() {
-		LdesConfig ldesConfig = new LdesConfig();
+		ldesConfig = new LdesConfig();
```

Comment:
I see there was already a ldesConfig defined before in tests.. your change seems logical, but did it work before?

Reply:
Yes because the entire config setup is done here and then never modified in the tests. I will modify my code to be consistent with this